glossary.yml

```
@@ -6057,6 +6057,11 @@
     def: >
       المُتغير هو موقع يٌخزِن البيانات المؤقتة داخل البرنامج ويمكن تعديله وتخزينه وعرضه
       عند الحاجة.
+  ja:
+    term: "変数 (プログラム)"
+    def: >
+      プログラム内で使用される、データが関連付けられる箱のようなものの総称。変数の値は定義した後で変更が可能。
+      変数は英語で「variable」(ヴァリアブル)という。
```

```suggestion
      変数は英語で「variable」(ヴァリアブル)という。
```Delete the sentence.

---

line removed 
```suggestion
```